lib/vanagon/engine/base.rb

```diff
@@ -10,7 +10,7 @@ def initialize(platform, target = nil)
         @platform = platform
         @required_attributes = ["ssh_port"]
         @target = target if target
-        @target_user = "root"
+        @target_user = @platform.target_user ? @platform.target_user : "root"
```

Comment:
It seems like if this is being added to the platform level, then "root" should be added as a default there. then this conditional can just go away.